src/collections/Footer/config.ts

```diff
@@ -0,0 +1,160 @@
+import { accessByTenant } from '@/access/byTenant'
+import { filterByTenant } from '@/access/filterByTenant'
+import { contentHashField } from '@/fields/contentHashField'
+import { tenantField } from '@/fields/tenantField'
+import { generatePreviewPath } from '@/utilities/generatePreviewPath'
+import type { CollectionConfig, TextFieldValidation } from 'payload'
+
+const validateHashtag: TextFieldValidation = (value: string | null | undefined): string | true => {
+  if (value === null || value === undefined) return true
+  else
+    return (
+      value?.match(/^#[A-Za-z0-9_](?:[A-Za-z0-9_]|(?:\.(?!\.))){0,28}[A-Za-z0-9_]$/)?.length ===
```

Comment:
Why `foo.match().length === 1` instead of `foo.test()`?

Reply:
No reason other than I've just always used `match`. Cleaned this area up a little